### PR TITLE
Add new test case "testGetLines" for lucene/core/analysis/WordlistLoader

### PR DESCRIPTION
**Pull request by hack4chang**
## Description
- Add new test case to test the ```getLines``` in ```WordlistLoader``` functions normally.
I create a string testcase that contains comment lines and blank lines to test ```getLines``` method. In order to pass a InputStream, I converted string into byteArray and then into byteArrayInputStream. This testcase proved that the function works well. By the definition of the function itself, the comment lines would have '#' at the start of the line, but I'm not sure if a line start with some spaces and then the '#' sign appears after the indentation count as a comment line as well. TBD

### CODE REVIEW

File: lucene/core/src/test/org/apache/lucene/analysis/TestWordlistLoader.java [hunk 1]
```diff
@@ -77,4 +82,17 @@ public void testSnowballListLoading() throws IOException {
...
   }
+
+  public void testGetLines() throws IOException {
+    String s = "One \n#Comment \n \n Two \n  Three  \n";
+    Charset charset = StandardCharsets.UTF_8;
+    byte[] sByteArr = s.getBytes(charset);
+    InputStream sInputStream = new ByteArrayInputStream(sByteArr);
+    List<String> lines = WordlistLoader.getLines(sInputStream, charset);
+    assertEquals(3, lines.size());
+    assertFalse(lines.contains("#Comment"));
```
Comment (stefanvodita) [inline lucene/core/src/test/org/apache/lucene/analysis/TestWordlistLoader.java:93]: Nit: The other asserts, taken together, already make a stronger guarantee about this comment than this assert, making it a bit redundant.

**Review by stefanvodita — APPROVED** [review]

File: lucene/core/src/test/org/apache/lucene/analysis/TestWordlistLoader.java (hunk 1 above)
Comment (hack4chang) [reply lucene/core/src/test/org/apache/lucene/analysis/TestWordlistLoader.java:93]: Hi, yeah sounds resonable, I will make a change.

Comment (stefanvodita) [reply lucene/core/src/test/org/apache/lucene/analysis/TestWordlistLoader.java]: Maybe I wasn't clear, but I only referred to the assertion about an item missing from the list, while the new commit also removed the assertion about the length of the list. If we assert length 3 and then assert on 3 elements of the list, that fully characterises the list, right?

Comment (hack4chang) [reply lucene/core/src/test/org/apache/lucene/analysis/TestWordlistLoader.java]: Ah, got it. Yeah you're right. I'll add the length assertion back.

**Review by stefanvodita — APPROVED** [review]
Thank you for persevering with my nitpicky comments!

**Pull request merged**